src/components/checkSelect.tsx

```diff
@@ -68,8 +68,11 @@ const CheckSelect: React.FC<CheckSelectProps> = ({
                 </svg>
               </div>
             </div>
-            <div className="text-body-small">
-              <label htmlFor="comments" className="text-primary-text">
+            <div className="text-sm/6">
+              <label
+                htmlFor={checkSelection.id}
+                className="font-medium text-gray-900"
+              >
```

Comment:
Same issue as previous PR regarding styling, can change here and close the old PR.